Objects/obmalloc.c

```diff
@@ -103,6 +109,39 @@ _PyMem_RawFree(void *ctx, void *ptr)
     free(ptr);
 }
 
+static void *
+_PyMem_RawAlignedAlloc(void *ctx, size_t alignment, size_t size)
+{
+    if (size == 0) {
+        size = 1;
+    }
+
+#ifdef MS_WINDOWS
+    return _aligned_malloc(size, alignment);
+#else
+    void *ptr;
+
+    if (posix_memalign(&ptr, alignment, size) < 0) {
```

Comment:
As I understand the man page,  `posix_memalign()` can return other error codes, which are not necessarily negative.  I suggest `!= 0`.

Reply:
Fixed.